src/main/java/frc/trigon/robot/subsystems/collector/placeholdercollector/PlaceholderCollectorConstants.java

```diff
@@ -0,0 +1,110 @@
+package frc.trigon.robot.subsystems.collector.placeholdercollector;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.*;
+import frc.trigon.robot.constants.RobotConstants;
+
+public class PlaceholderCollectorConstants {
+    static final boolean FOC_ENABLED = true;
+    private static final int
+            COLLECTING_MOTOR_ID = 1,
+            ANGLE_MOTOR_ID = 2,
+            ENCODER_ID = 1;
+    private static final InvertedValue
+            COLLECTING_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive,
+            ANGLE_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue
+            COLLECTING_MOTOR_NEUTRAL_MODE = NeutralModeValue.Coast,
+            ANGLE_MOTOR_NEUTRAL_MODE = NeutralModeValue.Coast;
+    private static final AbsoluteSensorRangeValue ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final SensorDirectionValue ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double ENCODER_OFFSET = 0;
+    private static final double ROTOR_TO_SENSOR_RATIO = 1;
```

Comment:
ROTOR_TO_SENSOR_RATIO is just gear ratio. Since this is also used for the sim, have one constant in the CollectorConstants. It's also better practice to have the collection gear ratio configured in the SensorToMechanism variable.